### ISSUE TITLE
Malware Detection.

### ISSUE THREAD

**Issue by NefariousIntent2** [thunderstorm010/ShowHiddenChannels#6]
Attempts to start a process called 'chrome.exe'. Unsure if this is a malware. But stay the fuck away until we find a cause

**Comment by thunderstorm010** [comment]
I didn't change any inner code from DevilBro's plugin.

**Comment by thunderstorm010** [comment]
I modified **only** metadata and update links from this commit
https://github.com/mwittrien/BetterDiscordAddons/tree/b510da9b3ad72ba3b773f7d79e391a459f31732d

**Comment by NefariousIntent2** [comment]
> I modified **only** metadata and update links from this commit https://github.com/mwittrien/BetterDiscordAddons/tree/b510da9b3ad72ba3b773f7d79e391a459f31732d

LMAO liar

**Comment by thunderstorm010** [comment]
Yea what lines i have changed, please tell me

**Comment by thunderstorm010** [comment]
Closing this issue for now. If anyone else has any problems, please create another issue and tell differences.

**Comment by LDzik** [comment]
from what I have checked there is no malware (in plugin or in library)

**Comment by TollyH** [comment]
I have checked the diffs of every file from the original source (plugin and library) and none of the changes are even remotely malicious. Only metadata and URLs have been updated, and the URLs all point to this public repo.
Not sure how you even "ran" the script in a VM and reliably monitored it seeing as it's a plugin not a standalone, but at any rate I can't reproduce this so-called "chrome.exe" launch.
